Cargo.toml

```diff
@@ -14,12 +14,17 @@ include = [
     "Cargo.toml",
 ]
 
+[features]
+async = []
+async_test = ["async", "futures-executor"]
```

Comment:
Seems we can remove the `async` features altogether now?

Reply:
Oh, I removed it and then cherry-picked it back again.